frontend/src/lib/components/Support/SupportModal.tsx

```diff
@@ -40,11 +40,17 @@ const SUPPORT_TICKET_KIND_TO_PROMPT: Record<SupportTicketKind, string> = {
     support: 'What can we help you with?',
 }
 
-export function SupportModal({ loggedIn = true }: { loggedIn?: boolean }): JSX.Element {
+export function SupportModal({ loggedIn = true }: { loggedIn?: boolean }): JSX.Element | null {
     const { sendSupportRequest, isSupportFormOpen } = useValues(supportLogic)
     const { setSendSupportRequestValue, closeSupportForm } = useActions(supportLogic)
     const { objectStorageAvailable } = useValues(preflightLogic)
 
+    if (!preflightLogic.values.preflight?.cloud) {
+        if (isSupportFormOpen) {
+            lemonToast.error(`Support via modal isn't provided for self-hosted instances.`)
```

Comment:
nit I'd say support form or support tickets but 🚢

Reply:
Hmm, it _was_ a bit clunky, I think the most to-the-point naming for this is gonna be "in-app support"